src/local-time.lisp

```diff
@@ -562,17 +562,21 @@ In other words:
                  :sec (sec-of timestamp)
                  :day (day-of timestamp)))
 
+(defun sec-day-subtimezone (sec day timezone)
```

Comment:
I'd like more explanation as to why having separate sec/day functions is an improvement. You've simplified the outer functions at the cost of dragging the logic out of them. Is this to avoid allocating timestamps?

If these new functions are meant for internal use, they should be prepended with a `%`. If not, they should be exported by the package and documented.

Reply:
The other comments are on PR 1 and PR2 when I realized it was just splitting up the review.